products/health-care/digital-health-modernization/mhv-to-va.gov/medical-devices-and-supplies/incident-reponse/playbook.md

```diff
@@ -0,0 +1,95 @@
+
+### Issue Investigation Steps
+
+This section outlines the steps to investigate and resolve issues related to the Medical Supplies Reorder.
+
+#### Initial Assessment
+
+- **Identify the Issue:** Use Datadog alerts to identify the issue. Determine whether it's an API failure, a frontend error, or a performance bottleneck.
+- **Impact Analysis:** Assess the impact of the issue. Determine how many users are affected and the severity of the impact on the feature's functionality.
+
+#### Detailed Investigation
+
+- **Log Analysis:** Review Datadog logs to pinpoint the error's origin. Look for stack traces, error messages, and the conditions under which the error occurred.
+- **Reproduction:** Attempt to reproduce the issue in a test environment based on the information from Datadog and user reports.
+- **Cross-Reference Metrics:** Use Datadog to cross-reference performance metrics around the time the issue occurred. This can help identify if the issue is isolated or part of a larger systemic problem.
+
+#### Resolution and Follow-Up
+
+- **Fix Implementation:** Once the issue is identified and understood, develop a fix. This could involve code changes, configuration updates, or infrastructure adjustments.
+- **Testing and Deployment:** Test the fix thoroughly in a staging environment before deploying to production.
+- **Monitoring Post-Fix:** Closely monitor Datadog after deploying the fix to ensure the issue is resolved and no new issues have arisen.
+- **Postmortem Analysis:** Conduct a postmortem analysis to understand the root cause and document learnings. Update the playbook accordingly to improve response for future incidents.
+
+### Flipper Features and Rollback
+
+- Flipper for the API endpoint
+- Note: We will be in direct contact with all 13 pilot participants during the pilot phase. Any issues that arise, we will be able assist.
+
+## Security
+
+<!--  descibe any security concerns the responders should be aware, for example: Does your product have PII? Do you log sensitive information that needs to be handled in a particular manner? Does your product have a known security vulnerability that has been accepted by leadership? etc. -->
+
+No PII/PHI captured in this feature.
+
+## Incident Response Steps
+
+- React/comment on the monitor alert in Slack to let the team know you are aware and investigating the issue
+- Review a list of recent monitor alerts, including other related platforms such as downstream services, va.gov platform issues, login.gov, etc.
+- Check out the [Dashboard](https://vagov.ddog-gov.com/dashboard/dfy-bgk-try/medications-device-ordering-pod)
+- Reach out to OCTO stakeholders in the [#vagov-supply-reordering](https://dsva.slack.com/archives/C05DFSM57FW) slack channel
+- Begin to troubleshoot the error and root cause
+  - If needed, begin coordination with the downstream API team in [#va-cto-supply-reordering-alerts](https://dsva.slack.com/archives/C07BSFDCK2B)
```

Comment:
Here's a link to the Upstream data source teams chat: https://teams.microsoft.com/l/chat/19:f0db76db670d4a93881bc64945aacd05@thread.v2/conversations?context=%7B%22contextType%22%3A%22chat%22%7D